vars/workflow-config-map-hidden.env

```diff
@@ -0,0 +1,8 @@
+workflowArtifactRepositoryConfigMapExecutorImage=argoproj/argoexec:v2.4.2
+workflowArtifactRepositoryConfigMapContainerRuntimeExecutor=docker
+workflowArtifactRepositoryConfigMapKeyFormat=artifacts/{{workflow.namespace}}/{{workflow.name}}/{{pod.name}}
+workflowArtifactRepositoryConfigMapAccessKeySecretName=onepanel
+workflowArtifactRepositoryConfigMapAccessKeySecretKey=argoArtifactRepositoryS3AccessKeyValue
+workflowArtifactRepositoryConfigMapSecretKeySecretName=onepanel
+workflowArtifactRepositoryConfigMapSecretKeySecretKey=argoArtifactRepositoryS3SecretKeyValue
```

Comment:
Same with `argoArtifactRepositoryS3SecretKeyValue`, this should be simply called `artifactRepositoryS3AccessKeyValue` as these can be used elsewhere S3 is needed.